mlflow/models/output_parsers.py

```diff
@@ -0,0 +1,60 @@
+from dataclasses import asdict
+from typing import Any, Dict
+
+from mlflow.models.rag_signatures import (
+    ChainCompletionChoice,
+    ChatCompletionResponse,
+    Message,
+    StringResponse,
+)
+from mlflow.utils.annotations import experimental
+
+try:
```

Comment:
Can we move these under `mlflow/langchain`? We should be able to remove `try-catch` for import then.

Reply:
we dont want users to import these from the `mlflow.langchain` namespace. these are output parsers that go along with `mlflow.models.rag_signatures`, so we want to put these in `mlflow.models` as well

Reply:
Hmmm - @B-Step62 has a point here. Why not `mlflow.langchain.BaseTransformOutputParser`? These are *langchain* output parsers.

Reply:
IMO this is not blocking feedback. Approving the PR.